test/write.spec.ts

```diff
@@ -96,6 +96,7 @@ const gitHubContext = {
     payload: {
         repository: {
             private: false,
+            html_url: 'https://github.com/user/repo',
```

Comment:
Could you please provide an additional test for a URL different than the GitHub one?

Reply:
Added [test for GHES](https://github.com/mjochum/github-action-benchmark/blob/fix/serverUrls/test/write.spec.ts#L1224)